app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+
+  def recent_posts
```

Comment:
Kindly rename this method to `recent_comments` get the last 5 comments and not posts.

Reply:
This change has been done well